blueprints/ember-frost-tabs/index.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '^0.29.0'}
+        {name: 'ember-frost-core', target: '^0.29.0'},
+        {name: 'ember-simple-uuid', target: '0.1.4'}
```

Comment:
Ok i understand your point, just a note to be careful with blueprints. You probably noticed that but if you are putting ^1.0.0 in blueprints you will get the latest matching this in your package.json, but you will get an exact version (no ^ or ~ or any other semver special characters)